features/vendor_views_closed_auction.feature

```diff
@@ -13,6 +13,8 @@ Feature: Vendor views closed auctions
     And I should see when the auction started
     And I should see when the auction ended
     And there should be meta tags for the closed auction
+    And an email notification is sent to the winning bidder
+    And I should receive an email notifying me that I won
```

Comment:
another option would be to use Timecop to travel to the time that we send the emails...but not sure if clock would run in a test env